generators/entity-server/templates/src/test/java/package/service/dto/EntityDTOTest.java.ejs

```diff
@@ -22,9 +22,13 @@ import org.junit.jupiter.api.Test;
 import static org.assertj.core.api.Assertions.assertThat;
 import <%= packageName %>.web.rest.TestUtil;
 <%_
+let primaryKeyType = pkType;
 let hasOauthUser = false;
 for (idx in relationships) {
+    const otherEntityName = relationships[idx].otherEntityName;
+    const isUsingMapsId = relationships[idx].useJPADerivedIdentifier;
```

Comment:
I would have named it `usingMapsId` instead, as it's a variable and not a function

Reply:
ok I update that

Reply:
Thanks!